django/test/runner.py

```diff
@@ -764,11 +765,11 @@ def is_discoverable(label):
     return os.path.isdir(os.path.abspath(label))
 
 
-def reorder_suite(suite, classes, reverse=False):
+def reorder_tests(tests, classes, reverse=False):
```

Comment:
I know that it's a part of undocumented private API, but folks can use it so I would add [release notes](https://docs.djangoproject.com/en/dev/releases/4.0/#miscellaneous):
```
* The undocumented ``django.test.utils.reorder_suite()`` function is renamed to ... and 
```

Reply:
Thanks, @felixxm! Done.